sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1237,6 +1237,10 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
     entry.get(conf)
   }
 
+  def get(entry: String): Option[String] = {
```

Comment:
Related to the "RapidsConf is not SparkConf" comment, I'm not a fan of adding this API. It encourages an undesired behavior.

Reply:
This is not resolved.  Like I said, RapidsConf does not necessarily mean SparkConf, and this API encourages that assumption.